src/PgNamed.hs

```diff
@@ -189,6 +191,27 @@ queryNamed conn qNamed params =
     withNamedArgs qNamed params >>= \(q, actions) ->
         liftIO $ PG.query conn q (toList actions)
 
+{- | Queries the database with a given row parser, query, and named parameters
+and expects a list of rows in return.
+
```

Comment:
Could you also add documentation on why somebody may want to use `queryWithNamed` and not just `queryNamed`?

Reply:
Done this~